account_brand/readme/USAGE.md

```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
+
+1.  Go to Accounting \> Customers \> Invoices
+2.  Select or create an invoice
+3.  Enter the information and select the brand
+4.  Print the PDF report. It includes the information of the brand.
```

Comment:
I don't see point 4 implemented. Is it? @sbejaoui

Reply:
@GuillemCForgeFlow 
I think that point 4 is https://github.com/OCA/brand/pull/205/files

To use this module, you need to:

1. Go to Settings > Users & Companies > Brands
2. Add a new brand or select an existing one
3. Enter brand information and select the a layout
4. Go to any branded object abd print the PDF report. It includes the
   information of the brand.

Reply:
Can you rebase for merge it?